app/main.go

```diff
@@ -96,9 +98,61 @@ func main() {
 		log.Fatalf("Invalid flags: %v", err)
 	}
 
-	http.HandleFunc("/", index)
-	http.HandleFunc("/sync", sync)
-	http.HandleFunc("/cleanup", cleanup)
+	config := tsbridge.NewConfig(&tsbridge.ConfigOptions{
+		Filename:                 *metricConfig,
+		MinPointAge:              *minPointAge,
+		CounterResetInterval:     *counterResetInterval,
+		SDLookBackInterval:       *sdLookBackInterval,
+		SDInternalMetricsProject: *sdInternalMetricsProject,
+		UpdateParallelism:        *updateParallelism,
+		UpdateTimeout:            *updateTimeout,
+		EnableStatusPage:         *enableStatusPage,
+		StorageEngine:            *storageEngine,
+		SyncPeriod:               *syncPeriod,
+		SyncCleanupAfter:         *syncCleanupAfter,
+	})
+
+	h := web.NewHandler(config)
+
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		web.Index(w, r, config)
```

Comment:
Should `Index` just be part of `web.Handler` as well? I don't think it gets called independently now.

Reply:
Fixed